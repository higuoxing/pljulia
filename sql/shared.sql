-- test the shared data
create function setme(key text, val text) returns void as $$
global GD[key] = val
$$ language pljulia;

create or replace function getme(key text) returns text as $$
return GD[key]
$$ language pljulia;

select setme('mykey', 'myval');

select getme('mykey');

drop function setme;
drop function getme;