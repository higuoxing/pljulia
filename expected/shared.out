-- test the shared data
create function setme(key text, val text) returns void as $$
global GD[key] = val
$$ language pljulia;
create or replace function getme(key text) returns text as $$
return GD[key]
$$ language pljulia;
select setme('mykey', 'myval');
 setme 
-------
 
(1 row)

select getme('mykey');
 getme 
-------
 myval
(1 row)

drop function setme;
drop function getme;
